portfolio/src/main/webapp/galleryScript.js

```diff
@@ -0,0 +1,38 @@
+function openGallery() {
+	document.getElementById('myGallery').style.display = 'block';
+}
+
+function closeGallery() {
+	document.getElementById('myGallery').style.display = 'none';
+}
+
+var slideIndex = 1;
+showSlidingGallery(slideIndex);
+
+function moveNSlides(n) {
+	showSlidingGallery(slideIndex += n);
+}
+
+function showCurrentSlide(n) {
+	showSlidingGallery(slideIndex = n);
+}
+
+function showSlidingGallery(n) {
+	var i;
+	var slides = document.getElementsByClassName('slide');
+	var descriptionText = document.getElementById('description');
+  var slideNumber = document.getElementById('slide-number');
+	if (n > slides.length) {
+		slideIndex = 1;
+	}
+	if (n < 1) {
+		slideIndex = slides.length;
+	}
+	for (i = 0; i < slides.length; i++) {
+		slides[i].style.display = 'none';
+	}
+	slides[slideIndex - 1].style.display = 'block';
+    descriptionText.style.fontSize = 'x-large';
+	descriptionText.innerHTML = slides[slideIndex - 1].getElementsByTagName('img')[0].alt;
+    slideNumber.innerHTML = slideIndex + '/' + slides.length;
```

Comment:
The indentation has gone a bit funny here and elsewhere in this function - worth just tidying it up to keep everything neat and tidy :-)

```suggestion
        slideNumber.innerHTML = slideIndex + '/' + slides.length;
```

Reply:
Honestly I am not sure what happened, in my local file looks good

Reply:
No worries these things happen :)